Add more sample clients for examples/local.

enisoc opened the pull request:
@michael-berlin 

These are alternative clients that show how to do the same thing as examples/local/client.py in other languages. This came about because a user got stuck on the fact that php/demo.php was incompatible with examples/local.

michael-berlin commented on examples/local/client.go:
```diff
@@ -0,0 +1,111 @@
+
+	// Read it back from the master.
+	fmt.Println("Reading from master...")
+	rows, err := db.Query("SELECT id,msg FROM test_table")
```
Put a space after the comma?

id, msg

michael-berlin commented on examples/local/client.go:
```diff
@@ -0,0 +1,111 @@
+	}
+	defer dbr.Close()
+
+	rows, err = dbr.Query("SELECT id,msg FROM test_table")
```
id, msg

michael-berlin commented on examples/local/client.go:
```diff
@@ -0,0 +1,111 @@
+	flag.Parse()
+
+	// Connect to vtgate.
+	connStr := fmt.Sprintf(`{"protocol": "grpc", "address": "%s", "keyspace": "%s", "shard": "%s", "tablet_type": "%s", "streaming": %v, "timeout": %d}`,
```
lol, nice

You're already using "keyspace" and "shard". That was fast ;-)

michael-berlin commented:
LGTM

enisoc commented:
I addressed the comments, and also added the new clients to the local_example test, so Travis should fail if you forget to fix this as part of #1378. :)

michael-berlin commented:
LGTM for additional changes.

Feel free to merge it :)
